src/core/Akka.MultiNodeTestRunner/Program.cs

```diff
@@ -221,25 +238,41 @@ static void Main(string[] args)
                     {
                         foreach (var test in discovery.Tests.Reverse())
                         {
-                            if (!string.IsNullOrEmpty(test.Value.First().SkipReason))
+                            var node = test.Value.First();
+                            if (!string.IsNullOrEmpty(node.SkipReason))
                             {
-                                PublishRunnerMessage($"Skipping test {test.Value.First().MethodName}. Reason - {test.Value.First().SkipReason}");
+                                PublishRunnerMessage($"Skipping test {node.MethodName}. Reason - {node.SkipReason}");
                                 continue;
                             }
 
                             if (!string.IsNullOrWhiteSpace(specName) &&
-                                CultureInfo.InvariantCulture.CompareInfo.IndexOf(test.Value.First().TestName,
+                                CultureInfo.InvariantCulture.CompareInfo.IndexOf(node.TestName,
                                     specName,
                                     CompareOptions.IgnoreCase) < 0)
                             {
-                                PublishRunnerMessage($"Skipping [{test.Value.First().MethodName}] (Filtering)");
+                                PublishRunnerMessage($"Skipping [{node.MethodName}] (Filtering)");
+                                continue;
+                            }
+
+                            var fullName = $"{node.TypeName}+{node.MethodName}";
```

Comment:
Full name of the test

Reply:
Why not just do one or the other? Each test class usually only has a single multinodefact due to XUnit's insistence on it